biogrid/libs/biogrid-simulator/src/lib/building/building.ts

```diff
@@ -1,11 +1,19 @@
 import {
   EnergyUser,
-  Battery,
   ItemPosition,
   Distance,
   Energy,
 } from '@biogrid/grid-simulator';
-import { GRID_ITEM_NAMES, BUILDING, RESISTANCE } from '../config';
+import { BUILDING, RESISTANCE } from '../config';
+
+export interface BuildingParams {
+  energy: number,
+  x: Distance,
+  y: Distance,
+  gridItemName: string,
+  readonly minCapacity?: Energy,
+  readonly maxCapacity?: Energy,
```

Comment:
I don't understand the value of setting these as readonly, since this is just an interface for the params, not the Building class itself

Reply:
I thought that it would be the same as the one for the building, I will address this

Reply:
> I don't understand the value of setting these as readonly, since this is just an interface for the params, not the Building class itself

I thought that it would be the same as the one for the building, I will address this